recipes/xkcd-cli/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "xkcd-cli" %}
+{% set version = "0.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/xkcd-cli-{{ version }}.tar.gz
+  sha256: ee48592bfaedb1f3284c30e9b413e101c12aaf027fcbb312897116996d528497
+
+build:
+  entry_points:
+    - xkcd = src.cli:cli
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - click
+    - requests
+    - pillow
+
+test:
+  imports:
+    - src
+  commands:
+    - pip check
+    - xkcd --help
+  requires:
+    - pip
+
+about:
+  license: MIT
+  license_file: LICENSE
```

Comment:
Looks like [this should be packaged in the future]( https://github.com/dotcs/xkcd-cli/commit/1d6f82fae4a9531c9d030b468315708a94af99ec#diff-50c86b7ed8ac2cf95bd48334961bf0530cdc77b5a56f852c5c61b89d735fd711R11-R13 ). Asked about a new release here ( https://github.com/dotcs/xkcd-cli/issues/6 )

Nothing to do here. Just noting for awareness